nav2_bringup/launch/nav2_simulation_launch.py

```diff
@@ -16,207 +16,156 @@
 
```

Comment:
Can we rename this file tb3_simulation_launch.py? Since this has specific references to the turtlebot3 packages, that seems more clear.

Reply:
Sure, changed it.